[gamemodes]/[play]/play/play_players.lua

```diff
@@ -0,0 +1,35 @@
+local spawnsCount = #playerSpawns
+local skinsCount = #playerSkins
+
+function playSpawnPlayer(playerElement)
+	if isElement(playerElement) then
+		local randomSpawn = math.random(1, spawnsCount)
+		local spawnData = playerSpawns[randomSpawn]
+		local posX, posY, posZ, rotX = spawnData[1], spawnData[2], spawnData[3], spawnData[4]
+		local randomSkin = math.random(1, skinsCount)
```

Comment:
The 1 is unnecessary.
```suggestion
		local randomSkin = math.random(skinsCount)
```